spec/factories/info_requests.rb

```diff
@@ -83,6 +83,12 @@
         end
       end
 
+      factory :not_held_request do
+        after(:create) do |info_request, evaluator|
```

Comment:
👍 on the feedback, but given no cop has failed I don't think there's anything to address here.

We shoukd

* Make a PR enabling the specific cop for this
* Make an issue for enabling `.rubocop_todo.yml` so that the idea doesn't get lost when this PR is merged (I did have a look at this, but need to take a bit of time to understand)

Reply:
:+1:

Reply:
Added issue https://github.com/mysociety/alaveteli/issues/4649